packages/adena-extension/src/components/approve/approve-adding-network/approve-adding-network-table/approve-adding-network-table.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { RecoilRoot } from 'recoil';
+import { ThemeProvider } from 'styled-components';
+import { render } from '@testing-library/react';
+import theme from '@styles/theme';
+import { GlobalStyle } from '@styles/global-style';
+import ApproveAddingNetworkTable, { ApproveAddingNetworkTableProps } from './approve-adding-network-table';
+
+describe('ApproveAddingNetworkTable Component', () => {
+  it('ApproveAddingNetworkTable render', () => {
+    const args: ApproveAddingNetworkTableProps = {
+      name: 'Teritori Testnet',
+      rpcUrl: 'https://testnet.gno.teritori.com:26658/',
+      chainId: 'teritori-3',
```

Comment:
teritori rpc `https://testnet.gno.teritori.com:26658/status?` returns `teritori-1`

Reply:
changed to empty.